From 4b8ed3c0c1e67977bf46cba3625c80cf6aab0805 Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <salimma@fedoraproject.org>
Date: Thu, 6 Jul 2023 16:21:45 -0500
Subject: [PATCH] [cpp2] also compile the new IOUringUtil.cpp source

Signed-off-by: Michel Alexandre Salim <salimma@fedoraproject.org>
---
 thrift/lib/cpp2/CMakeLists.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/thrift/lib/cpp2/CMakeLists.txt b/thrift/lib/cpp2/CMakeLists.txt
index 2494da3574..6f32b6f9d6 100644
--- a/thrift/lib/cpp2/CMakeLists.txt
+++ b/thrift/lib/cpp2/CMakeLists.txt
@@ -244,6 +244,7 @@ add_library(
   server/Cpp2Connection.cpp
   server/Cpp2Worker.cpp
   server/CPUConcurrencyController.cpp
+  server/IOUringUtil.cpp
   server/LoggingEvent.cpp
   server/LoggingEventHelper.cpp
   server/MonitoringMethodNames.cpp
-- 
2.41.0

