From 934da73ea8b3b491b6b79f6062c07586941d4f27 Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <michel@michel-slm.name>
Date: Wed, 4 Nov 2020 10:05:57 -0800
Subject: [PATCH] Fix static build

Signed-off-by: Michel Alexandre Salim <michel@michel-slm.name>
---
 thrift/compiler/CMakeLists.txt | 1 -
 1 file changed, 1 deletion(-)

diff --git a/thrift/compiler/CMakeLists.txt b/thrift/compiler/CMakeLists.txt
index c0e134cc6..a82ea06d0 100644
--- a/thrift/compiler/CMakeLists.txt
+++ b/thrift/compiler/CMakeLists.txt
@@ -145,7 +145,6 @@ add_executable(
   compiler.cc
   mutator/mutator.cc
   validator/diagnostic.cc
-  validator/validator.cc
   ast/visitor.cc
 )
 target_link_libraries(
-- 
2.28.0

