diff -Naur fbthrift-2020.11.09.00.orig/thrift/lib/py3/CMakeLists.txt fbthrift-2020.11.09.00/thrift/lib/py3/CMakeLists.txt
--- fbthrift-2020.11.09.00.orig/thrift/lib/py3/CMakeLists.txt	2020-11-13 11:02:55.921912284 -0800
+++ fbthrift-2020.11.09.00/thrift/lib/py3/CMakeLists.txt	2020-11-13 11:01:51.945242774 -0800
@@ -89,7 +89,11 @@
     DEPENDS thrift-symlinks
   )
 
-  python_add_module(${_module_name} "${_cxx}")
+  if(${_f} STREQUAL "types")
+    python_add_module(${_module_name} "${_cxx}" "enums.cpp")
+  else()
+    python_add_module(${_module_name} "${_cxx}")
+  endif()
   target_link_libraries(${_module_name} thriftcpp2)
   set_target_properties(${_module_name}
     PROPERTIES
diff -Naur fbthrift-2020.11.09.00.orig/thrift/lib/py3/setup.py fbthrift-2020.11.09.00/thrift/lib/py3/setup.py
--- fbthrift-2020.11.09.00.orig/thrift/lib/py3/setup.py	2020-11-08 04:22:54.000000000 -0800
+++ fbthrift-2020.11.09.00/thrift/lib/py3/setup.py	2020-11-13 10:59:06.399510339 -0800
@@ -45,6 +45,7 @@
     Extension("thrift.py3.client", sources=["thrift/py3/client.so"]),
     Extension("thrift.py3.server", sources=["thrift/py3/server.so"]),
     Extension("thrift.py3.ssl", sources=["thrift/py3/ssl.so"]),
+    Extension("thrift.py3.reflection", sources=["thrift/py3/reflection.so"]),
 ]
 setup(
     name="thrift",
