diff -Naur fbthrift-2021.07.20.00.orig/thrift/lib/py3/CMakeLists.txt fbthrift-2021.07.20.00/thrift/lib/py3/CMakeLists.txt
--- fbthrift-2021.07.20.00.orig/thrift/lib/py3/CMakeLists.txt	2021-07-19 17:12:02.000000000 -0700
+++ fbthrift-2021.07.20.00/thrift/lib/py3/CMakeLists.txt	2021-07-28 10:26:54.480713170 -0700
@@ -56,7 +56,9 @@
                          reflection.pxd
                          reflection.pyx
                          serializer.pxd
-                         serializer.pyx)
+                         serializer.pyx
+                         stream.pxd
+                         stream.pyx)
 
 file(WRITE ${CMAKE_CURRENT_BINARY_DIR}/cybld/thrift/__init__.pxd)
 
@@ -72,6 +74,7 @@
   "thrift/py3/server.cpp"
   "thrift/py3/ssl.cpp"
   "thrift/py3/reflection.cpp"
+  "thrift/py3/stream.cpp"
 )
 foreach(_mod ${cpp_tgts})
   get_filename_component(_f ${_mod} NAME_WE)
