From 594417e713bc248c65e3577a25df9799fce03963 Mon Sep 17 00:00:00 2001
From: Michel Alexandre Salim <michel@michel-slm.name>
Date: Tue, 10 Nov 2020 00:09:44 -0800
Subject: [PATCH] Honor DESTDIR when installing Python extension on Unix
 systems

Signed-off-by: Michel Alexandre Salim <michel@michel-slm.name>
---
 thrift/lib/py3/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/thrift/lib/py3/CMakeLists.txt b/thrift/lib/py3/CMakeLists.txt
index bdd27e71b..c9c9e1332 100644
--- a/thrift/lib/py3/CMakeLists.txt
+++ b/thrift/lib/py3/CMakeLists.txt
@@ -101,12 +101,17 @@ endforeach()
 include_directories(${PYTHON_INCLUDE_DIRS})
 
 # Build and Install Thrift Python Bindings using Python setuptools
+if (UNIX)
+  set(ROOT_ARG "--root \$DESTDIR/")
+endif ()
+
 install(CODE "
   execute_process(COMMAND
   python3 ${CMAKE_CURRENT_SOURCE_DIR}/setup.py bdist_wheel
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/cybld)
   execute_process(COMMAND
   python3 ${CMAKE_CURRENT_SOURCE_DIR}/setup.py install
+  --prefix ${CMAKE_INSTALL_PREFIX} ${ROOT_ARG}
   WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/cybld)"
 )
 
-- 
2.28.0

